### ISSUE TITLE
[1.6] InvoiceServ: Unknown Error Code: 67

### ISSUE THREAD

**Issue by mihato** [PayPal-Mobile/ios-here-sdk-dist#64]
I'm trying to process payment with manually entered card data. Here is my code:

```objective-c
PPHAmount *total = [PPHAmount amountWithString:[NSString stringWithFormat:@"%.2f", self.amount] inCurrency:self.currency];
[PayPalHereSDK activeMerchant].currencyCode = self.currency;
PPHTransactionManager *transactionManager = [PayPalHereSDK sharedTransactionManager];
transactionManager.ignoreHardwareReaders = YES;
[transactionManager beginPaymentWithInvoice:[[PPHInvoice alloc] initWithItem:@"item" forAmount:total]];
[transactionManager setManualEntryOrScannedCardData:manualCardData];
    
[transactionManager beginPaymentWithAmount:total andName:@"simplePayment"];
transactionManager.manualEntryOrScannedCardData = manualCardData;
    
[transactionManager processPaymentWithPaymentType:ePPHPaymentMethodKey
                        withTransactionController:nil
                                completionHandler:^(PPHTransactionResponse *record) 
{
    if (record.error ) {
        NSLog(@"FAIL");
    }
    else {
        NSLog(@"SUCCESS");
    }
}];
```

PPH console log:

```
> PPH Info: PayPalHere: Clearing payment data
> PPH Warning: PayPalHere: 
--- _OBSCURO_EMFJSONResponse https://www.sandbox.paypal.com/webapps/hereapi/merchant/v1/invoices

Headers: {
  "Strict-Transport-Security" : "max-age=14400",
  "Server" : "Apache",
  "X-Cnection" : "close",
  "Paypal-Debug-Id" : "919b2588c2efb",
  "Content-Encoding" : "gzip",
  "Set-Cookie" : "SPARTAJSESSIONID=5701aa40b2b27; Domain=.paypal.com; Path=\/; Secure; HttpOnly, SPARTAJSESSIONID=92357001b1c42; Domain=.paypal.com; Path=\/; Secure; HttpOnly, SPARTAJSESSIONIDV2=zU7rL1E8Bsax4qQTPMB6.x4Ipk8yQFa9Hfw.rlCwN3ENsPtSgzdP0Zi9jVmsqG3rNHGso2zGRfc; Domain=.paypal.com; Path=\/; Secure; HttpOnly, X-PP-SILOVER=name%3DSANDBOX3.WEB.1%26silo_version%3D880%26app%3Dhereapispartaweb%26TIME%3D3347520342; domain=.paypal.com; path=\/; Secure; HttpOnly, X-PP-SILOVER=; Expires=Thu, 01 Jan 1970 00:00:01 GMT",
  "Vary" : "Accept-Encoding,Authorization",
  "Content-Type" : "application\/json;charset=utf-8",
  "Content-Length" : "127",
  "Date" : "Mon, 12 Oct 2015 10:13:27 GMT"
}

Error: (null)

Data: {"correlationId":"919b2588c2efb","errorCode":600000,"message":"InvoiceServ: Unknown Error Code: 67","errorType":"service_error"}

> PPH Error: PayPalHere: Ending payment with error. Error Domain=service_error Code=600000 The operation couldn’t be completed. (service_error error 600000.) apiMessage=InvoiceServ: Unknown Error Code: 67 message=(null) correlationId=919b2588c2efb @ 2015-10-12 10:13:28 +0000
```

Any ideas what code 67 means?

**Comment by ppmtscory** [comment]
@mihato it looks like you are using an invalid currency.  Can you ensure that what you are using for currency matches up to the merchant account's currency?  For example, if you are using a US merchant account, then the currency should be set to USD.

**Comment by mihato** [comment]
Thank you for response. You are right, problem is in currency code. Now it's working but I have one more question. Is it possible to accept payments via PayPal Here in different currencies? All I found so far is that currency conversion should be done automatically. After I created ``PPHAmount`` object in different currency I had a successful payment transaction. But in pending transactions report of my sandbox account this transaction displayed in primary currency.

```objective-c
[PayPalHereSDK activeMerchant].currencyCode = @"USD";
PPHAmount *total = [PPHAmount amountWithString:@"5.00" inCurrency:@"EUR"];
```
Pending transaction report:
![paypal-pending-transaction](https://cloud.githubusercontent.com/assets/937623/10479906/b292e5b6-726f-11e5-89bc-15fc82e72b75.png)
Same is in sandbox notifications on developer portal.
